third_party/intel/lib/TritonGENToLLVM/TritonGENToLLVMPass.cpp

```diff
@@ -1415,8 +1372,50 @@ struct TritonMatrix2DBlockPrefetchLowering
   LogicalResult
   matchAndRewrite(TritonGEN::Matrix2DBlockPrefetchOp op, OpAdaptor adaptor,
                   ConversionPatternRewriter &rewriter) const override {
-    LLVM::CallOp callOp = createGenISA2DBlockPrefetch(op, rewriter);
-    rewriter.replaceOp(op, callOp);
+    // TODO: Remove GenISA lowering after PoC productization is completed.
+    char *env = std::getenv("TRITONGEN_FORCE_GENISA");
+    const bool useGenISA = env ? (bool)std::atoi(env) : false;
```

Comment:
```suggestion
    const bool useGenISA = std::getenv("TRITONGEN_FORCE_GENISA");
```

Maybe we can just do this if the variable's defined?

Reply:
Good idea.

Reply:
@victor-eds with the suggested change `TRITONGEN_FORCE_GENISA=0` would make `useGenISA` to be true, which is not what we want, so reverted the change.